src/test/docker/alpine/Dockerfile

```diff
@@ -38,7 +39,8 @@ RUN apk add \
     jq \
     aspell \
     aspell-en \
-    valgrind
+    valgrind \
+    openssh
```

Comment:
FWIW, all tests on this branch pass in my current image which doesn't have ssh installed. It would be good to know what exactly was failing as maybe a different kind of test fix is needed.

Reply:
Oh that's interesting. Maybe it was a symptom of one of the other issues, I'll see if I can pull it back up.